web/src/routes/(user)/search/+page.svelte

```diff
@@ -44,6 +44,8 @@
 	});
 
 	$: term = $page.url.searchParams.get('q') || data.term || '';
+	$: is_metadata_search = $page.url.searchParams.get('clip') === 'false';
+	$: term = `${!is_metadata_search && term !== '' ? 'm:' : ''}${term}`;
```

Comment:
Here is the error I got when trying to run this 
![image](https://github.com/immich-app/immich/assets/27055614/de742f40-c6bd-4bd6-9018-921bd82a6450)

Reply:
I see. This is my first time with reactive statements; I just figured it was a false warning since the typescript compiled properly. I just switched to an anonymous function with better legibility too.